--- PackageInstallerActivityOriginal.java	2024-12-26 23:35:30.682568894 +0000
+++ PackageInstallerActivity.java	2024-12-25 12:43:17.720667750 +0000
@@ -143,9 +143,15 @@
 
         viewToEnable.setVisibility(View.VISIBLE);
 
-        mEnableOk = true;
-        mOk.setEnabled(true);
+        mEnableOk = false;
+        mOk.setEnabled(false);
         mOk.setFilterTouchesWhenObscured(true);
+
+        if ("com.luddite.app.store.INSTALL_PACKAGE".equals(getIntent().getAction())) {
+            mEnableOk = true;  // Enable install button for our store
+            mOk.setEnabled(true);
+        }
+
     }
 
     /**

